extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -210,7 +210,8 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
       const height = this.element.getAttribute('height');
       const width = this.element.getAttribute('width');
-      if (height != ADSENSE_RSPV_WHITELISTED_HEIGHT) {
+      // height is set to 0 by amp-auto-ads to avoid reflow.
+      if (height != 0 && height != ADSENSE_RSPV_WHITELISTED_HEIGHT) {
```

Comment:
@keithwrightbos to make sure height=0 has no special meaning in AdSense.

Reply:
We do not send the ad request if the measured area of the slot is 0 (see https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/amp-a4a.js#L560) otherwise it has no special meaning

Reply:
Are we OK to resolve this?

Reply:
Yep I think so